backend/de.metas.business/src/main/java/de/metas/invoice/service/impl/CreditMemoInvoiceCopyHandler.java

```diff
@@ -63,8 +50,12 @@
 {
 	private final InvoiceCreditContext creditCtx;
 	private final BigDecimal openAmt;
+
+	@SuppressWarnings("unused")
 	private final String trxName;
```

Comment:
`why do we need this when it's unused? pls remove or add a quick comment to explain

Reply:
This was added when I thought we should keep the code commented out. And ofc I didn't see it when I deleted the code, because it had a suppress warning 🤦 

Thanks for finding it. Cleaned up now.